docs/reference/dropbox.md

```diff
@@ -0,0 +1,118 @@
+# Dropbox Connector
+
+The [Elastic Dropbox connector](../connectors/sources/dropbox.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+## Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+To use this connector as a **connector client**, use the **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+## Get a Refresh Token
+
+### Create Dropbox OAuth App
+
+In order to authenticate via the Dropbox package, the user needs to configure an app key, app secret and refresh token to fetch the data from Dropbox. 
+
+To generate App Key and App Secret, user needs to follow below steps:
+1. Register a new app in the [App Console](https://www.dropbox.com/developers/apps) and select Full Dropbox API app and choose the following required permissions.
+    - files.content.read
+    - sharing.read
+
+2. Once the app is created, users can have an **app key** and **app secret** on the settings tab.
+
+### Generate a Refresh Token
+
+To generate refresh token, user needs to follow below steps:
+1. Hit the following URL with the generated APP_KEY on the browser:
+    ```shell
+    https://www.dropbox.com/oauth2/authorize?client_id=<APP_KEY>&response_type=code&token_access_type=offline
+    ```
+    
+    User will get an **authorization code** by hitting the URL and that will be used to generate a refresh token.
```

Comment:
> User will get an **authorization code** by hitting the URL and that will be used to generate a refresh token.

In the HTTP response, you will get an **authorization code** that will be used to generate a refresh token.